<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analisis_demo</class>
 <widget class="QDialog" name="analisis_demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">RadioButton {
	text: &quot;Round X, -90 degr.&quot;
	ButtonGroup.group: tabRotationGroup
	Layout.preferredHeight: 15
	indicator.height: 100
	indicator.width: 10
	onClicked:{ currentRotationIndex = 2 }
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Home .png);
background-repeat: none;
background-position: center;
border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>691</width>
     <height>431</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>180</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>180</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>225</y>
      <width>180</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>225</y>
      <width>180</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="aleta1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#aleta1
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Aleta-izq.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="aleta2" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>56</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#aleta2
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Aleta-Dere.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="aleta3" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#aleta3
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Aleta-izq.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="aleta4" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#aleta4
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Aleta-Dere.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="left_menu" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#left_menu
{
background-color: rgb(19,19,50);
}</string>
   </property>
   <widget class="QPushButton" name="graph_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#graph_button
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Graph Menu.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#close_menu
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Close Menu.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="show_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#show_menu
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Menu.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="datewidget" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>5</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="control_A000">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="setpoint_presion">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>39</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#radioButton::indicator {
    width:   30px;
    height:  30px;
  
}</string>
   </property>
   <property name="text">
    <string>Setpoint Presion</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>% Valvula</string>
   </property>
  </widget>
  <widget class="QLabel" name="valv_ma">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>139</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>mA</string>
   </property>
  </widget>
  <zorder>show_menu</zorder>
  <zorder>frame</zorder>
  <zorder>left_menu</zorder>
  <zorder>pushButton</zorder>
  <zorder>datewidget</zorder>
  <zorder>control_A000</zorder>
  <zorder>setpoint_presion</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>valv_ma</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
